ci/scripts/integration_arrow.sh

```diff
@@ -30,15 +30,23 @@ gold_dir=$arrow_dir/testing/data/arrow-ipc-stream/integration
 : ${ARROW_INTEGRATION_JAVA:=ON}
 : ${ARROW_INTEGRATION_JS:=ON}
 
+echo "::group::Integration: Prepare: Archery"
+set -x
 pip install -e $arrow_dir/dev/archery[integration]
+set +x
+echo "::endgroup::"
 
+echo "::group::Integration: Prepare: Dependencies"
+set -x
```

Comment:
I don't know if it's possible to have some kind of shell script function to factor these out?

Reply:
Hmm. I couldn't create enough convenient functions as I imagined...